src/spikeinterface/curation/auto_merge.py

```diff
@@ -230,12 +202,9 @@ def get_potential_auto_merge(
                 "knn",
                 "quality_score",
             ]
-
-    for step in steps:
-        if step in _required_extensions:
-            for ext in _required_extensions[step]:
-                if not sorting_analyzer.has_extension(ext):
-                    raise ValueError(f"{step} requires {ext} extension")
+    if force_copy and compute_needed_extensions:
+        # To avoid erasing the extensions of the user
+        sorting_analyzer = sorting_analyzer.copy()
```

Comment:
Now this might not be needed anymore. This was because at the beginning of this function, one could use a sorting analyzer with already precomputed extension and recompute some with new params for the merges. But this is not possible anymore. Either extensions have been already computed with custom params, either we use default. So this copy is not needed anymore I guess

Reply:
We can leave that here in case user would not want to save some extra extensions that might have been needed during the merges